components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -45,17 +45,11 @@ const WorkspacesPage: FunctionComponent = () => {
 
     const { filteredActiveWorkspaces, filteredInactiveWorkspaces } = useMemo(() => {
         const filteredActiveWorkspaces = activeWorkspaces.filter(
-            (info) =>
-                `${info.workspace.description}${info.workspace.id}${info.workspace.contextURL}${info.workspace.context}`
```

Comment:
is it ok that we drop .context? not sure

Reply:
We have no `context` in new workspace shape

Reply:
yes, we don't want to expose everything, the question is whether it is important here? if not then it is fine

Reply:
Added `cloneUrl` and `branch` to search